roles/ffh.gw-statuspage/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+- name: Install python3, git
+  apt:
+    update_cache: yes
+    cache_valid_time: 3600
+    name: "{{ packages }}"
+  vars:
+    packages:
+    - python3
+    - git
+
+- name: Create auto user
+  register: createuser
+  user:
+    name: auto
+    system: yes
+    generate_ssh_key: yes
+
+- name: Create site directory
+  file:
+    state: directory
+    dest: /var/cache/gw-statuspage/site
+    owner: auto
+    recurse: yes
+
+- name: Clone site
+  become: yes
+  become_user: auto
+  git: >
+    repo=https://github.com/freifunkh/site.git
+    dest=/var/cache/gw-statuspage/site
+    accept_hostkey=true
+  register: site
+
+- name: Add aiyions package-repo into sources list
+  ansible.builtin.apt_repository:
+    repo: deb http://aiyionpri.me/repo/ ./
+    state: present
+    filename: aiyionprime
```

Comment:
Sounds a bit generic. Will this deb be the only file you are serving on that repo?

Reply:
Unlikely, but for now only each new version of gw-statuspage will be there.